drivers/media/i2c/Makefile

```diff
@@ -115,6 +115,7 @@ obj-$(CONFIG_VIDEO_S5C73M3)	+= s5c73m3/
 obj-$(CONFIG_VIDEO_ADP1653)	+= adp1653.o
 obj-$(CONFIG_VIDEO_LM3560)	+= lm3560.o
 obj-$(CONFIG_VIDEO_LM3646)	+= lm3646.o
+obj-$(CONFIG_VIDEO_LT6911UXC) += lt6911uxc.o
```

Comment:
The other lines have a TAB character before the `+=`.

Reply:
resolved in last commit